spec/ics-3-connection-semantics/header_tracking.tex

```diff
@@ -0,0 +1,33 @@
+\include{common}
+
+\begin{document}
+\begin{varwidth}{\linewidth}
+
+\begin{algorithmic}
+\Struct[Datagram]{CONNTRACK}
+\Field{Identifier}{identifier}
+\Field{Header}{header}
+\EndStruct
+\end{algorithmic}
+
+\begin{algorithmic}
+\Function{handleDatagram}{datagram}
```

Comment:
Nit: We probably want to make this consistent across ICSs (compare: #37, e.g., https://github.com/cosmos/ics/pull/37/files#diff-c443c461a2596896f59401c4087cc983 function vs procedure)

Reply:
Yes, let's standardize on "Function" - updated ICS 18.